src/circle_factory.py

```diff
@@ -33,4 +33,4 @@ def circle_from_list(list: list):
         raise ValueError("Elements must be lists containing three elements")
     if not true_for_all(lambda elem: isinstance(elem, float) or isinstance(elem, int), list):
         raise ValueError("All elements of list must be numbers")
-    return circle_factory(Point(list[0], list[1]), list[2])
+    return circle_factory(Point2D(list[0], list[1]), list[2])
```

Comment:
Er der en fordel ved at bruge point2d over point? 
Grunden til at vi valgte point,så vidt jeg husker, var at den var nem at skalere til 3d engang.

Reply:
Altså ikke umiddelbart andet end at man ved der er 2 værdier i den

Reply:
Tror point giver god mening hvis vores funktioner er "generiske" og altså behandler dem som "points af n dimensioner". Men vi behandler dem kun som 2d points.

Reply:
Men idk, med circle factory, det var bare for at bruge samme type alle steder 🤷‍♂️